ArduPlane/sensors.cpp

```diff
@@ -65,6 +65,10 @@ void Plane::accel_cal_update() {
  */
 void Plane::read_airspeed(void)
 {
+    static int airspeed_failed_count=0;
```

Comment:
please put variables in a structure in the Plane class, instead of using statics

Reply:
Ended up moving them to the airspeed state, so they are not tied to the vehicle code, it keeps them contained in the airspeed library.